Assets/TanksDemo/Assets/TanksInputActions.inputactions

```diff
@@ -48,21 +48,20 @@
                     "processors": "",
                     "interactions": "Press(behavior=2)",
                     "bindings": []
+                },
+                {
+                    "name": "Pause",
+                    "id": "020abfe6-6083-46ec-9487-3aa2ecb330f3",
+                    "expectedControlLayout": "",
```

Comment:
Strongly recommend setting action types / expectedControlLayouts on actions. Without them, the rebinding apparatus has no idea what it's looking for and just blindly filters for the most likely input.

Reply:
fixed